harvest/utils.py

```diff
@@ -246,6 +246,7 @@ def datetime_utc_to_local(date_time: dt.datetime, timezone: ZoneInfo) -> dt.date
     Converts a datetime object in UTC to local time, represented as a
     timezone naive datetime object.
     """
+    date_time = date_time.to_pydatetime()
```

Comment:
@tfukaza For Kraken, Alpaca, and Polygon, the `astimezone` function call fails because `date_time` is of type `pd.DatetimeIndex` and not of `datetime`. Not sure if Robinhood or the streamer is different.

Reply:
I think other DataFrames also has pd.DatetimeIndex. I might be wrong but I think pd.DatetimeIndex uses an older way to manage timezones, so we need to convert it to datetime.